privacyidea/static/components/config/views/config.smsgateway.edit.html

```diff
@@ -146,17 +159,4 @@ <h3 translate>Headers</h3>
         </tr>
     </table>
     </div>
-
-    <!-- button -->
-    <div class="text-center"
-         ng-show="checkRight('smsgateway_write')">
-        <button ng-click="createSMSgateway()"
-                id="createGatewayButton"
-                ng-disabled="formGatewayAdd.$invalid"
-                class="btn btn-primary">
-            <span class="glyphicon glyphicon-plus"></span>
-            <span translate>Save SMS gateway</span>
-        </button>
-    </div>
-
```

Comment:
For policies and events we have the "Save" button above _and_ below the options, maybe we should stick with that.

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.